test/unit/history/invoke.test.js

```diff
@@ -0,0 +1,23 @@
+import History from '../../../src/history'
+import Microcosm from '../../../src/microcosm'
+
+describe('History::invoke', function() {
+
+  it('it does invoke a repo that no longer exists', function() {
+    let parent = new Microcosm()
+    let child = parent.fork()
+
+    parent.on('change', () => child.teardown())
+
+    jest.spyOn(child, 'release')
+
+    parent.addDomain('test', {
+      getInitialState() {
+        return false
+      }
+    })
+
+    expect(child.release).not.toHaveBeenCalled()
+  })
```

Comment:
Still waiting for some details on that, however I was investigating and found this edge case, so I went ahead and stamped it out.

So we'll see :)